index.html

```diff
@@ -6,15 +6,15 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Things Gateway - Skill Setup</title>
+    <title>WebThings Gateway - Skill Setup</title>
```

Comment:
nit: the order here should probably be reversed, i.e. “Skill Setup - WebThings Gateway”.

Reply:
Done.